### ISSUE TITLE
`Lint/UselessConstantScoping` provides false positives when private_constant takes a list of arguments

### ISSUE THREAD

**Issue by corsonknowles** [rubocop/rubocop#14182]
--------

## Expected behavior

The 2nd, 3rd, etc argument to private_constant (which supports a list) should not raise an offense:

```ruby
    HTTP_MAX_REDIRECTS = 5
    HTTP_REDIRECT_CODES = T.let([301, 302, 307, 308].freeze, T::Array[Integer]) # rubocop:disable Layout/PrivateConstant
    private_constant :HTTP_MAX_REDIRECTS, :HTTP_REDIRECT_CODES
```

## Actual behavior

Have to disable the cop for false positive, or refactor to make multiple method calls to private_constant:
```ruby
    HTTP_MAX_REDIRECTS = 5
    private_constant :HTTP_MAX_REDIRECTS
    HTTP_REDIRECT_CODES = T.let([301, 302, 307, 308].freeze, T::Array[Integer])
    private_constant :HTTP_REDIRECT_CODES
```
## Steps to reproduce the problem

Code provided!

## RuboCop version

ruby//3.4.0/
/rubocop-1.75.2

**Comment by viralpraxis** [comment]
There's no `Layout/PrivateConstant` cop, did you mean `Layout/ClassStructure` or `Lint/UselessConstantScoping`? 🤔

**Comment by corsonknowles** [comment]
Thank you for clarifying!

I meant:

Lint/UselessConstantScoping

I found this issue when upgrading from our in house rule to the new
upstream equivalent from 1.72

I will update the Issue so it’s clearer for other readers

**Comment by corsonknowles** [comment]
Ah, thank you so much @viralpraxis -- this issue is actually incorrect. 

`Lint/UselessConstantScoping` solves the issue we were having rather than creates it.
